ktor-client/ktor-client-darwin/darwin/src/io/ktor/client/engine/darwin/DarwinWebsocketSession.kt

```diff
@@ -188,18 +188,18 @@ internal class DarwinWebsocketSession(
     }
 }
 
-private suspend fun NSURLSessionWebSocketTask.receiveMessage(
-): NSURLSessionWebSocketMessage = suspendCancellableCoroutine {
-    receiveMessageWithCompletionHandler { message, error ->
-        if (error != null) {
-            it.resumeWithException(DarwinHttpRequestException(error))
-            return@receiveMessageWithCompletionHandler
-        }
-        if (message == null) {
-            it.resumeWithException(IllegalArgumentException("Received null message"))
-            return@receiveMessageWithCompletionHandler
-        }
+private suspend fun NSURLSessionWebSocketTask.receiveMessage(): NSURLSessionWebSocketMessage =
```

Comment:
could we get to the old formatting? it looks less nested and more readable

Reply:
it fails on CI